lib/mailinator/models/email.rb

```diff
@@ -7,31 +7,21 @@ module Models
     class Email < Base
       def transform_data
         {
-          id: @data['data']['id'],
+          from_full: @data['data']['fromfull'],
+          date: DateTime.parse(@data['data']['headers']['date']),
+          received: @data['data']['headers']['received'],
+          from: @data['data']['headers']['from'],
+          to: @data['data']['headers']['to'],
           subject: @data['data']['subject'],
+          request_id: @data['data']['requestId'],
           body: @data['data']['parts'].first['body'],
-          body_html: retrieve_body_html,
```

Comment:
Any special reason for removing `f17d5b837f674eada25fe9e3e85cf262`?

Reply:
It appears the JSON has been updated to no longer have separate body and body_html elements. just the 'body'